src/Diagnostics.DataProviders/DataProviderConfigurations/IMdmDataProviderConfiguration.cs

```diff
@@ -24,5 +24,10 @@ public interface IMdmDataProviderConfiguration
         /// Gets or sets monitoring account.
         /// </summary>
         string MonitoringAccount { get; set; }
+
+        /// <summary>
+        /// Gets or sets if monitoring account is regional.
+        /// </summary>
+        bool IsRegional { get; set; }
```

Comment:
@enihcam , The location header is not currently present in Applens (when CSS uses the tool) so they will not be able to pass the location header. Also, an addition of this property will mean that we update our config too and our deployment pipeline.

I looked at the rest of code and seems like the location is being used to identify the correct MDM endpoint depending on the location. Can this be handled in the GIST that we discussed about instead of letting the platform construct the MDM endpoint. Also, if the MDM endpoint is constructed inside the gist, each partner can create their own endpoints based on their logic instead of depending on the platforms logic.

I'd suggest removing this property from the backend and let the gist take care of creating the correct MDM endpoint.